pkgs/build-support/setup-hooks/separate-debug-info.sh

```diff
@@ -34,7 +34,7 @@ _separateDebugInfo() {
         # firmware blobs in QEMU.)
         (
             $OBJCOPY --only-keep-debug "$i" "$dst/${id:0:2}/${id:2}.debug"
-            $STRIP --strip-debug "$i"
+            $STRIP --strip-unneeded "$i"
```

Comment:
Will this ever be used on macOS? FWIW it supports neither flags.

strip is an area I've noticed where we tend to use non portable flags, maybe it's time to introduce a more bespoke strip wrapper…

Reply:
OK, all this sounds more like future improvements.